client/data/deposits/hooks.ts

```diff
@@ -186,3 +186,22 @@ export const useInstantDeposit = (
 
 	return { deposit, inProgress, submit };
 };
+
+export const useStandardDeposit = (
```

Comment:
This is very similar to the `useInstantDeposit` hook. I'm tempted to merge them into a single hook. However, I decided that it is best to stick to the "one function for one task" paradigm, rather than merge and use a param e.g. `depositType: 'instant' | 'standard'`.